crypto/rand/drbg_lib.c

```diff
@@ -1031,19 +1031,85 @@ static int drbg_bytes(unsigned char *out, int count)
     return ret;
 }
 
+/*
+ * Calculates the minimum length of a full entropy buffer
+ * which is necessary to seed (i.e. instantiate) the DRBG
+ * successfully.
+ *
+ * NOTE: There is a copy of this function in drbgtest.c.
```

Comment:
This makes me nervous.  Could we lose the static here, delete the copy in drbgtest.c and statically link the drbgtest executable?  The symbol won't be exported from libcrypto, the test will gain access and the duplication is gone.

Reply:
Of course I tried to to reuse the function from drbg_lib.c in the tests. In fact, I thought I had read in some place that the tests were linked statically. After this proved wrong I did not want to export a private symbol, so I thought adding a warning comment would be the best compromise. 

Changing the test framework to link the tests statically is out of my scope, so one of you would have to do that.

Reply:
@paulidale thanks for your hints regarding build.info, with your explanation the change was really trivial, see #7462.